core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java

```diff
@@ -253,15 +257,22 @@ public boolean delete(Supplier<JournalContext> journalContext, AlluxioURI uri) {
    */
   public String getMountPoint(AlluxioURI uri) throws InvalidPathException {
     String path = uri.getPath();
-
+    String candidatePath = ROOT;
```

Comment:
NIT: rename to `lastMount` or `deepestMount`?

Reply:
done